project-service/src/middleware/EurekaConnect.js

```diff
@@ -0,0 +1,81 @@
+const axios = require('axios').default;
```

Comment:
Lets simplify the idea. We will need two class here.

1. EruekaClient 

interface EurekaClient {
 mount() // calls when server start up
unMount() // calls when server shut down
sendHeartBeat() // send heartBeat in regular intervals
}

2. LoadBalancer 

LoadBalancer {
// need to maintain in-memory map when you are getting instances from Eureka
   getServiceInstances() // calls when server start up first and after in regular intervals
   sendRequest(LoadBalancingStrategy, serviceName) {
          LoadBalancingStrategy.loadBalance(serviceName)
   }// send Http request for service
}
LoadBalancingStrategy {
   loadBalance(serviceName)
}

RoundRobbinStrategy implements LoadBalancingStrategy {}